git-backdate

```diff
@@ -82,32 +91,34 @@ def rewrite_history(
     elif no_business_hours:
         min_hour = 18
         max_hour = 23
-    duration = len(days)
-    commits_per_day = math.ceil(len(commits) / duration)
-    last_timestamp = None
-    commit_count = len(commits)
-    day_progress = 0
-    for commit_index in range(commit_count):
-        progress = (commit_index + 1) / commit_count
+    prev_timestamp = prev_commit_timestamp(commits)
```

Comment:
If the previous timestamp is significantly off our start, this will screw over all preceding calculations re: commits per day.